packages/web3-eth-abi/package.json

```diff
@@ -41,6 +41,7 @@
 		"jest": "^27.3.1",
 		"jest-extended": "^1.1.0",
 		"jest-when": "^3.4.1",
+    "@humeris/espresso-shot": "^4.0.0",
```

Comment:
I don't think it's such a great idea to use such a young and unused library for our tests. Having something is better than nothing, but I've created [an issue](https://github.com/ChainSafe/web3.js/issues/4635) to revisit this

Reply:
This is quite a new topic to tests types so why we don't have any library. I had reviewed all the code of that library before using it, so good to go for now.